controller_definition.py

```diff
@@ -436,23 +436,33 @@ def adjustMixer(self, knob: int, dataType: str, action: str, selectedTrack: int)
         # Multiplies the trackGroup to 8 to get the index of the first track of that group
         trackFirst = trackGroup * 8
 
+        # If you receive a sensitivity value, set it to a controllable value, else default to 1
+        if (sensitivity):
+            sensitivity = sensitivity / 15
+
+            # When decreasing the mixer knob, the amount that it changes is dramatically different than increasing.
+            if (action == "DECREASE"):
+                sensitivity = sensitivity / 10
+        else:
+            sensitivity = 1
+
         if (trackGroup == 15) and (knob == 6 or knob == 7): # Control 15th group exception
             return
 
         else:
             if dataType == "VOLUME":
                 if action == "INCREASE":
-                    mixer.setTrackVolume(trackFirst + knob, mixer.getTrackVolume(trackFirst + knob) + config.KNOB_INCREMENTS_VOL)
+                    mixer.setTrackVolume(trackFirst + knob, mixer.getTrackVolume(trackFirst + knob) + (config.KNOB_INCREMENTS_VOL * sensitivity))
                 
                 elif action == "DECREASE":
-                    mixer.setTrackVolume(trackFirst + knob, mixer.getTrackVolume(trackFirst + knob) - config.KNOB_INCREMENTS_VOL)
+                    mixer.setTrackVolume(trackFirst + knob, mixer.getTrackVolume(trackFirst + knob) - (config.KNOB_INCREMENTS_VOL * sensitivity))
 
             elif dataType == "PAN":
                 if action == "INCREASE":
-                    mixer.setTrackPan(trackFirst + knob, mixer.getTrackPan(trackFirst + knob) + config.KNOB_INCREMENTS_PAN)
+                    mixer.setTrackPan(trackFirst + knob, mixer.getTrackPan(trackFirst + knob) + (config.KNOB_INCREMENTS_PAN * sensitivity))
```

Comment:
Offset should be `config.KNOB_INCREMENTS_PAN * log(speed, 63)` instead of `config.KNOB_INCREMENTS_PAN * sensitivity`. Same thing goes for decrement.